selenium/che-selenium-test/src/test/resources/projects/git-pull-conflict/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,15 @@
+<%--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+--%>
+<%
+   response.sendRedirect("spring/hello");
+%>
```

Comment:
Add line

Reply:
We should use new project in the resources for the test? May be use alredy existed?